pages/common/shellcheck.md

```diff
@@ -0,0 +1,19 @@
+# shellcheck
+
+> Shell script static analysis tool.
+
+- Check a shell script:
+
+`shellcheck myscript.sh`
```

Comment:
Let us be consistent and use `file.sh` everywhere. Also please wrap it in tokens (`{{}}`)

Reply:
Thanks. Addressed in new commit b97b8d8.